tests/js/merkleToken.scrypttest.js

```diff
@@ -33,10 +33,10 @@ describe("Test sCrypt contract merkleToken In Javascript", () => {
 
   const Token = buildContractClass(compileContract("merkleToken.scrypt"))
 
-  let token, lockingScriptCodePart, tx_
+  let token, lockingScriptCodePart, tx
 
   beforeEach(() => {
-    tx_ = new bsv.Transaction()
+    tx = newTx();
```

Comment:
Here we create an empty tx, so do not use `newTx()`. Same elsewhere.
`tx = new bsv.Transaction()`

Reply:
9a531461eca019f0a14ad12158cfe4e098d0e462